style/swift.md

```diff
@@ -1,3 +1,1037 @@
-# Swift style guide
+# iOS Style Guide (Swift)
 
-This document details our house style for writing Swift code.
+## Introduction
+
+This style guide is based on the [official raywenderlich.com Swift style guide](https://github.com/raywenderlich/swift-style-guide) but has some changes to better accommodate the practices of our team. We would like to thank [Ray and his team](https://www.raywenderlich.com), and all of those who contributed to their style guide, for creating it in the first place.
+
+Another source of style guidelines is the [API Design Guidelines](https://swift.org/documentation/api-design-guidelines/#include-words-to-avoid-ambiguity). These guidelines are also reflected in this document.
+
+Our primary goals, like theirs, are **clarity**, **consistency**, and **brevity** but also to achieve a sufficiently stable style guide to configure a linter from. Nonetheless, it's a living document that will evolve as our needs change, as well as alongside the Swift language.
+
+### Updates
+
+For some non-additive changes such as misspellings, grammar or compiler errors, changes to this guide can be performed by simply issuing a pull request. If case the changes are deemed more "controversial", open an issue to clarify the change and to allow discussion within the team so that other members of our team can participate in the discussion. We thank any and all such contributions in advance!
```

Comment:
```suggestion
For some non-additive changes such as misspellings, grammar or compiler errors, changes to this guide can be performed by simply issuing a pull request. In cases where the changes are deemed more "controversial", open an issue to clarify the change first, this will allow discussion within the team and other members of the team can participate in the discussion. We thank any and all such contributions in advance!
```